org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/IWorkspaceServiceParticipant.java

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Please add contributors in the header:

```
/*******************************************************************************
* Copyright (c) 2021 Red Hat Inc. and others.
* All rights reserved. This program and the accompanying materials
* which accompanies this distribution, and is available at
* http://www.eclipse.org/legal/epl-v20.html
*
* SPDX-License-Identifier: EPL-2.0
*
* Contributors:
*     Red Hat Inc. - initial API and implementation
*******************************************************************************/
```

Reply:
FWIW, it's been clarified by EMO that according to latest version of EDP those Contributors block are not at all mandatory for Eclipse projects.